letsencrypt/acme/challenges_test.py

```diff
@@ -236,37 +234,37 @@ def test_json_without_optionals(self):
 class ProofOfPossessionHintsTest(unittest.TestCase):
 
     def setUp(self):
-        jwk = other.JWK(key=KEY.publickey())
-        issuers = [
+        jwk = jose.JWKRSA(key=KEY.publickey())
+        issuers = (
             'C=US, O=SuperT LLC, CN=SuperTrustworthy Public CA',
             'O=LessTrustworthy CA Inc, CN=LessTrustworthy But StillSecure',
-        ]
-        cert_fingerprints = [
+        )
```

Comment:
I see how the tuples work, but it isn't what I would expect... The lists seem more appropriate.  My guess is that the lists caused problems with json serialization/deserialization???